### PR TITLE
Switch focus to going back to truly open source, rather than a foundation

### PR DESCRIPTION
**Pull request by brikis98**
See the ["avoiding war"](https://gruntwork-io.slack.com/archives/C05MAK37AFQ/p1692036211930699) thread for context.

This is a different approach / request that we can consider. I'm not sure if this is actually better, so we should discuss, and pick whether we like this approach or the one that's live now.

### CODE REVIEW

**Comment by coryodaniel** [comment]
Not only do I appreciate these edits, but I appreciate you calling out that this isn’t to start a war. <3

File: index.html [hunk 1]
```diff
@@ -66,13 +66,45 @@ <h1>The OpenTF Manifesto</h1>
...
-          to impartial bodies (the Linux Foundation and the Cloud Native Computing Foundation, respectively) who
-          can ensure that those building blocks can form solid and predictable underpinnings for our industry.
+          We believe that the essential building blocks of the modern Internet, such as Linux, Kubernetes, 
+          and Terraform need to be truly open source: that is the only way to ensure
+          that we are building our industry on top of solid and predictable underpinnings.
         </p>
 
         <p class="font-bold">
-          Our goal: ensure Terraform is handed over to an open source foundation.
+          Our goal: ensure Terraform remains truly open source—always.
```
Comment (coryodaniel) [inline index.html:75]:
```suggestion
          Our goal: ensure Terraform remains truly open-source — always.
```

**Comment by ZIJ** [comment]
+1 from Digger - foundation has no value in and of itself, it serves a purpose - to achieve what? to keep terraform with community. So cutting through that link directly makes sense.

File: index.html [hunk 2]
```diff
@@ -66,13 +66,45 @@ <h1>The OpenTF Manifesto</h1>
...
+        <p>
+          We ask HashiCorp to do the right thing by the community: instead of going forward with the
+          BSL license change, switch Terraform back to a truly open source license, and commit to keeping
+          it that way forever going forward. That way, instead of fracturing the community, we end up with 
+          a single, impartial, reliable home for Terraform where the whole community can unite to keep 
+          building this amazing ecosystem.
+        </p>
+
+        <p class="font-bold">
+          Our fallback plan: fork Terraform into a foundation.
```
Comment (orbitz) [inline index.html:99]: I believe this part should be removed.  I think the messaging should be:

1. HCP should move Terraform to an open source license.
2. If Terraform is given to a foundation, we have pledged support for the project.

I think this is positive and no threats.  I feel the threats play into HCP's messaging of people taking advantage.

**Comment by brikis98** [comment]
I just merged the latest from `main` into this branch.

File: index.html (hunk 1 above)
Comment (brikis98) [reply index.html:82]: The em dash shouldn't have spaces around it. As for "open source," we are pretty inconsistent with hyphenation, switching back and forth... Should probably rethink it globally and figure out how we want to do it.

File: index.html (hunk 2 above)
Comment (brikis98) [reply index.html:106]: Interesting idea! A few questions:

1. Is the implication that if HashiCorp makes it open source, but not in a foundation, we don't pledge any support?
2. I'll raise this up more broadly shortly, but does (1) imply that the signatories are _only_ signing on for the foundation piece... But not just making it open source? I suspect that's not the message we want to send. We want every company possible to sign this and show support for it...
3. And if HashiCorp does neither of these things... What then? In other words, if it's just a request, it sounds more like a _petition_ than a _manifesto_... Maybe that's OK, but I want to check.

**Review by SebastianStadil — APPROVED** [review]

**Pull request merged**